recipe/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "k-means-constrained" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e3cf1601ebe91c2317fd595399153e81c450085b15a072ea67f9850175a12102
+
+build:
+  # or-tools is missing on Windows due to OpenBLAS requirement 
+  skip: true  # [py<38 or win]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - wheel
+    - cython 0.29.33
+    - numpy 1.23.5
```

Comment:
Isn't 1.23 a little high? Wouldn't it break if this package is used with a lower version that we support (for example 1.21)?

Reply:
I messed up and went with the latest tag on their GH. What we actually want is v0.7.2. This applies to all of your comments, sorry for the waste of time. 

https://github.com/joshlk/k-means-constrained/blob/master/pyproject.toml

`cython>=0.29` is specified and this is the version we have with all archs and python variants.